src/Node/ModuleNode.php

```diff
@@ -170,7 +171,13 @@ protected function compileClassHeader(Compiler $compiler)
             ->raw(" extends Template\n")
             ->write("{\n")
             ->indent()
+            ->write("/**\n")
+            ->write(" * @var Source\n")
+            ->write(" */\n")
             ->write("private \$source;\n")
```

Comment:
What about using a proper type hint (this class is final anyway and the variable private)?

```suggestion
            ->write("private Source \$source;\n")
```